packages/component/src/BasicSendBox.js

```diff
@@ -66,8 +65,15 @@ BasicSendBox.propTypes = {
   })
 };
 
-export default connectToWebChat(({ dictateState, styleSet, webSpeechPonyfill }) => ({
-  dictationStarted: dictateState === STARTING || dictateState === DICTATING,
+// TODO: [P3] We should consider exposing core/src/definitions and use it instead
```

Comment:
file issue?

Reply:
I will leave the `TODO` inside the files for now. And we do a pass to copy all the TODO to GitHub Issues.